src/main/java/uk/gov/beis/els/categories/lamps/service/LampsService.java

```diff
@@ -141,4 +143,43 @@ public ProcessedEnergyLabelDocument generateHtml(LampsFormPackagingArrow form, L
         .asProcessedEnergyLabelLampsPackagingArrow(ProductMetadata.LAMPS_PACKAGING_ARROW, form);
   }
 
+  public ProcessedEnergyLabelDocument generateHtml(LampsPreSeptember2021ApiForm form,
+                                                   LegislationCategory legislationCategory) {
+    return new TemplatePopulator(templateParserService.parseTemplate("labels/lamps-light-sources/lamps.svg"))
+        .setCondensingMultilineText("supplier", form.getSupplierName())
+        .setCondensingMultilineText("model", form.getModelName()).setRatingArrow("rating",
+            RatingClass.valueOf(form.getEfficiencyRating()), legislationCategory.getPrimaryRatingRange())
+        .setText("kwh", form.getEnergyConsumption())
+        .asProcessedEnergyLabel(ProductMetadata.LAMPS_FULL, form);
```

Comment:
Same my other comment about endpoints and product metadata. Do we need to pass in the product metadata for the different endpoints? @jamesbarnett91